packages/playground/src/index.ts

```diff
@@ -0,0 +1,283 @@
+import { getGentraceApiKey } from "@gentrace/core";
+import { WebSocket } from "ws";
+import { v4 as uuidv4 } from "uuid";
+
+export { init } from "@gentrace/core"; // for accessing the Gentrace API key
+
+import { AsyncLocalStorage } from "async_hooks";
+const asyncLocalStorage = new AsyncLocalStorage();
+
+interface customObject {
```

Comment:
Let's use types not interfaces whenever possible. We want to be explicit about what key/value pairs are required. Typescript interfaces are much too permissive.

Reply:
[commit](https://github.com/gentrace/gentrace-node/commit/53b1aa986c43fd3d1aa18c0cad09af13112732cd)